src/resources/TeamContributionCalendar/TeamContributionCalendar.js

```diff
@@ -26,30 +26,40 @@ export default class TeamContributionCalendar {
   async renderBasicAppearance() {
     this.renderActualCalendar();
 
-    const defaultUserJsonCalendar = await GitHubUtils.getJsonFormattedCalendarSync(
-      this.configs.proxyServerUrl, DefaultUsers.GitHub,
-    );
+    try {
+      const defaultUserJsonCalendar = await GitHubUtils.getJsonFormattedCalendarSync(
+        this.configs.proxyServerUrl, DefaultUsers.GitHub,
+      );
 
-    const defaultUserEmptyCalendar = GitHubUtils.setEmptyCalendarValues(defaultUserJsonCalendar);
+      const defaultUserEmptyCalendar = GitHubUtils.setEmptyCalendarValues(defaultUserJsonCalendar);
 
-    this.updateCalendar({
-      contributions: 0,
-      updatedActualCalendar: defaultUserEmptyCalendar,
-    });
+      this.updateCalendar({
+        contributions: 0,
+        updatedActualCalendar: defaultUserEmptyCalendar,
+      });
+    } catch {
+      this.updateCalendar({
+        isLoading: false,
+      });
+
+      throw new Error('Could not load the calendar of the default user. The issue might be related to the cors-proxy server.');
```

Comment:
I'd propose to print the original message. `"Could not load the calendar of the default user. Error: #{error.message}"`. No need to mention specifics, like proxy.

Reply:
FYI, the `Error` keyword is already mentioned in the message by default. Would you still like to stick with the format you proposed above?

![Képernyőfotó 2019-04-23 - 8 43 04](https://user-images.githubusercontent.com/30864202/56560119-ddfe0e80-65a3-11e9-95fd-2f19a259417e.png)

Reply:
Ah I overlooked this part. So the error handling should happen around the request and ideally, it's not even a try-catch but checks whether the response returned success. Putting many things in a try-catch in generally a bad idea.